grails-app/controllers/au/org/ala/sds/SDSController.groovy

```diff
@@ -71,6 +106,7 @@ class SDSController {
     /**
      * SDS layers
      */
+
```

Comment:
Add an @Operation here too?

Reply:
Added. And also included in API Gateway Specs for sds-webapp APIs: https://github.com/AtlasOfLivingAustralia/aws-apig-cognito/blob/test/sds-webapp/app_setup/apis/app-rest.yaml  -> https://api.test.ala.org.au/sds-webapp/ws/layers